apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -17,11 +17,7 @@ import TeacherActionsBox from './TeacherActionsBox';
 
 import styles from './progress-table-legend.module.scss';
 
-export default function IconKey({
-  isViewingValidatedLevel,
-  expandedLessonIds,
-  sectionId,
-}) {
+export default function IconKey({sectionId}) {
```

Comment:
I think we should remove the extra properties wherever we make an IconKey, otherwise, everything looks great!

Reply:
Clarifying question: is the `sectionId` the extra prop you are referring to?  It looks like this prop is used to track user interaction with `AnalyticsReporter`.  I left that in because I assumed we still wanted to track that data here.